rpcs3/Emu/RSX/D3D12/D3D12GSRender.cpp

```diff
@@ -166,6 +167,7 @@ D3D12GSRender::D3D12GSRender()
 		// Adapter with specified name
 		if (adapter_name == desc.Description)
 		{
+			vram_size = desc.DedicatedVideoMemory != 0 ? desc.DedicatedVideoMemory : desc.DedicatedSystemMemory;
```

Comment:
Well, from msdn:
```
DedicatedVideoMemory
Type: SIZE_T
The number of bytes of dedicated video memory that are not shared with the CPU.
DedicatedSystemMemory
Type: SIZE_T
The number of bytes of dedicated system memory that are not shared with the CPU. This memory is allocated from available system memory at boot time.
SharedSystemMemory
Type: SIZE_T
The number of bytes of shared system memory. This is the maximum value of system memory that may be consumed by the adapter during operation. Any incidental memory consumed by the driver as it manages and uses video memory is additional.
```
So it might be some combination of the too.